publications.include

```diff
@@ -15,48 +15,45 @@
               Daniel Arndt,
               Wolfgang Bangerth,
               Bruno Blais,
-              Thomas C. Clevenger,
               Marc Fehling,
-              Alexander V. Grayver,
+              Rene Gassm&ouml;ller,
               Timo Heister,
               Luca Heltai,
+              Uwe K&ouml;cher,
               Martin Kronbichler,
               Matthias Maier,
               Peter Munch,
               Jean-Paul Pelteret,
-              Reza Rastak,
-              Ignacio Thomas,
+              Sebastian Proell,
+              Konrad Simon,
               Bruno Turcksin,
-              Zhuoran Wang,
-              David Wells
+              David Wells,
+              Jiaqi Zhang
               <br/>
-              <strong>The <abbr>deal.II</abbr> Library, Version 9.2
+              <strong>The <abbr>deal.II</abbr> Library, Version 9.3
               </strong>
               <br>
-	      Journal of Numerical Mathematics, vol. 28, pp. 131-146. 2020.
+	      Journal of Numerical Mathematics, accepted. 2021.
 	      <br>
-              <a href="https://doi.org/10.1515/jnma-2020-0043">DOI:
-                 10.1515/jnma-2020-0043</a>
+              <a href="https://doi.org/??">DOI:
+                 10.1515/jnma-???</a>
               <br>
-	      (<a href="https://dealii.org/deal92-preprint.pdf"
+	      (<a href="https://dealii.org/deal93-preprint.pdf"
                   target="_top">preprint</a>)
               <br>
               <pre>
-@article{dealII92,
-  title     = {The \texttt{deal.II} Library, Version 9.2},
+@article{dealII93,
+  title     = {The \texttt{deal.II} Library, Version 9.3},
   author    = {Daniel Arndt and Wolfgang Bangerth and Bruno Blais and
-               Thomas C. Clevenger and Marc Fehling and Alexander V. Grayver and
-               Timo Heister and Luca Heltai and Martin Kronbichler and Matthias Maier and
-               Peter Munch and Jean-Paul Pelteret and Reza Rastak and
-               Ignacio Thomas and Bruno Turcksin and Zhuoran Wang and David Wells},
+               Marc Fehling and Rene Gassm{\"o}ller and
+               Timo Heister and Luca Heltai and Uwe K{\"o}cher
+	       and Martin Kronbichler and Matthias Maier and
```

Comment:
Please adjust these slightly awkwardly indented lines -- these are shown verbatim on the page